com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/AdvancedOptionsUIBlock.cs

```diff
@@ -34,21 +47,29 @@ public class Styles
             public static readonly GUIContent motionVectorForVertexAnimationText = new GUIContent("Motion Vector For Vertex Animation", "When enabled, HDRP will correctly handle velocity for vertex animated object. Only enable if there is vertex animation in the ShaderGraph.");
         }
 
-        protected MaterialProperty specularOcclusionMode = null;
-        protected MaterialProperty addPrecomputedVelocity = null;
+        MaterialProperty specularOcclusionMode = null;
+        MaterialProperty addPrecomputedVelocity = null;
 
-        protected const string kSpecularOcclusionMode = "_SpecularOcclusionMode";
-        protected const string kAddPrecomputedVelocity = HDMaterialProperties.kAddPrecomputedVelocity;
+        const string kSpecularOcclusionMode = "_SpecularOcclusionMode";
+        const string kAddPrecomputedVelocity = HDMaterialProperties.kAddPrecomputedVelocity;
 
-        Expandable  m_ExpandableBit;
+        ExpandableBit  m_ExpandableBit;
         Features    m_Features;
 
-        public AdvancedOptionsUIBlock(Expandable expandableBit, Features features = Features.All)
+        /// <summary>
+        /// Construct the Advanced options material UI block.
+        /// </summary>
+        /// <param name="expandableBit">Bit index used to store the foldout state.</param>
+        /// <param name="features">Features of the block.</param>
+        public AdvancedOptionsUIBlock(ExpandableBit expandableBit, Features features = Features.All)
         {
             m_ExpandableBit = expandableBit;
             m_Features = features;
         }
 
+        /// <summary>
+        /// Use this function to load the material properties you need in your block.
```

Comment:
Loads the material properties for the block.

Reply:
Can you do this for the other LoadMeterialProperties summaries too? There are 16 others(I think) in this or. You can Find and Replace in your ide, but I can't Find and Add a Comment in git sadly!

Reply:
Sure!